_includes/version.html

```diff
@@ -22,10 +22,11 @@ <h4>Supported Platforms:</h4>
 <h4>Windows version:</h4>
 <h5 class="subtitle-mute">(Windows XP and later versions supported)</h5>
 <h5>
-  <a href="{{ latest_version_release_link }}/{{ version.win32_setup_file }}">32-bit Setup (XP+)</a>
+  <a href="{{ latest_version_release_link }}/{{ version.win32_setup_file }}">32-bit Setup (Vista+)</a>
   |
-  <a href="{{ latest_version_release_link }}/{{ version.win64_setup_file }}">64-bit Setup (Vista+)</a>
+  <a href="{{ latest_version_release_link }}/{{ version.win64_setup_file }}">64-bit Setup (7+)</a>
 </h5>
+<h5><a href="{{ latest_version_release_link }}/{{ version.winXP_setup_file }}">32/64-bit Setup (XP+)(temporary not released)</a></h5>
```

Comment:
```suggestion
<h5><a href="{{ latest_version_release_link }}/{{ version.winXP_setup_file }}">32/64-bit Setup (XP+) (currently unavailable)</a></h5>
```
I think the winxp installer would be a new thing introduced in the next release, and we should really add this along with the version bump for the next release instead, and only if #3988 gets in